tips/028_input_guardrails.md

```diff
@@ -0,0 +1,175 @@
+## Effective AI Engineering #28: Input Guardrails
+
+**Are malicious users draining your AI budget with expensive attacks?** Every prompt injection or jailbreak attempt hits your most expensive models, even when you could reject them at the gate.
```

Comment:
general comment, but we should create a bank of real world stories that we can create hooks around. Finding that stories, written with a good hook are much better.